src/register/UsernameField.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes, { string } from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { connect } from 'react-redux';
+
+import { Button } from '@edx/paragon';
+
+import { usernameSuggestionsSelector } from './data/selectors';
+import { FormGroup } from '../common-components';
+import messages from './messages';
+
+const UsernameField = (props) => {
+  const { intl, usernameSuggestions } = props;
+
+  return (
+    <FormGroup {...props}>
+      {usernameSuggestions && usernameSuggestions.length > 0 ? (
```

Comment:
if the default value is `[]` do we need to add the first check, shouldn't `usernameSuggestions.length` be enough?

Reply:
Yes, it should be.